bin/list-legacy-filenames

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo ".ecspresso-version"
```

Comment:
I didn't know this. It is convenient.

references:
https://asdf-vm.com/plugins/create.html#optional-scripts
https://asdf-vm.com/manage/configuration.html#home-asdfrc